webextensions/api/menus.json

```diff
@@ -177,7 +177,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤58"
+                  "version_added": "44"
```

Comment:
❓ Unclear, but the referenced Chrome 44 IMO doesn't provide evidence. Two options:
1. It was not available in Chrome 44 yet, because https://github.com/chromium/chromium/blob/44.0.2403.157/chrome/browser/extensions/api/context_menus/context_menus_api_helpers.h#L93-L99 checks for `is_platform_app()`, not `is_extension()`, but I found [the same in 59](https://github.com/chromium/chromium/blob/59.0.3071.135/chrome/browser/extensions/api/context_menus/context_menus_api_helpers.h#L93-L99), so it cannot be that.
2. It was available before Chrome 44, because [Chrome 38 already has this](https://github.com/chromium/chromium/blob/38.0.2123.0/chrome/browser/extensions/api/context_menus/context_menus_api_helpers.h#L89-L92). But also [Chrome 26 already has this](https://github.com/chromium/chromium/commit/f5ec20caaf9afe15af718dbf7e2116572000de24#diff-e48e634f5d5e22a07d0aa8722e02a3437d033ada4be238b6adea30b3e7fda1bfR82).